<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qceflibtest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>50</height>
         </size>
        </property>
        <widget class="QPushButton" name="pushButton1">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>0</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>发给page1</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>发送到js</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>201</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>30</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>接受Web数据：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_webmsg">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>30</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <zorder>pushButton1</zorder>
        <zorder>lineEdit</zorder>
        <zorder>label</zorder>
        <zorder>label_webmsg</zorder>
        <zorder>pushButton2</zorder>
       </widget>
      </item>
      <item>
       <widget class="QCefWebView" name="widget_1" native="true">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCefWebView" name="widget_2" native="true">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border:1px solid #555;</string>
        </property>
        <widget class="QLineEdit" name="lineEdit_url">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>281</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>http://www.baidu.com</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_loaduri">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>20</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>LoadUri</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">border:1px solid #555;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCefWebView</class>
   <extends>QWidget</extends>
   <header location="global">qcefwebview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
